gemfiles/rails52.gemfile

```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 
-gem "rails", ">= 5.2.0.rc1", "< 5.3"
+gem "rails", ">= 5.2.0.beta2", "< 5.3"
```

Comment:
This was automatically generated when I ran appraisal.

Reply:
Ahhh. Should I change it back?

Reply:
Looks like #409 missed updating the Appraisal file, fixed in 402c31ecd792ac52294bf1744660d7c72bea22fb